cnvlib/cmdutil.py

```diff
@@ -5,11 +5,20 @@
 from skgenome import tabio
 
 from .cnary import CopyNumArray as CNA
+from skgenome import GenomicArray as GA
 
-
-def read_cna(infile, sample_id=None, meta=None):
+# todo: check all invocations of read/read_cna and make sure the parameter is forwarded.
+def read_cna(infile, treat_par_on_chrx_as_autosomal_for_genome_build, into=CNA, sample_id=None, meta=None):
```

Comment:
I feel like the decision on PAR is important and I wanted to make sure one is forced to think about it when creating the `CNA`. At least here (with `read_cna()`) I could do this. When invoking the ctor directly (or even cast'ing), I cannot force the developer to take the decision on PAR.

I do understand your concern about breakage for anyone using `cnvlib` as dependency. I think in the current code base, nothing breaks for now. In any case, I'm happy to move the parameter to the end (`into`) as well and thus take a default decision. Just let me know.

Reply:
As for putting it into `meta`, yes, this is a good idea. This would allow to refactor the code and do the `if-else` here instead of this [line](https://github.com/etal/cnvkit/blob/0a1735753f505e5748d89e56f078cdd70177072c/cnvlib/cnary.py#L32).